src/cycax/cycad/engines/cycax_part_freecad.py

```diff
@@ -31,6 +31,16 @@
 BACK = "BACK"
 REAR = "BACK"
 
+nut_specifications = {  # This is a global variable that will be used to cut the nuts by the OpenSCAD engine.
```

Comment:
This is the FreeCAD file so comment is out of date.  If this is a copy of code from OpenSCAD then please refactor. This definition should only be in one place.

Reply:
Since this is the code being called in FreeCAD and I suspect you are redefining the Nut spec. Pass information like this from the main code via JSON to Application specific code like this.